pkg/proxy/register.go

```diff
@@ -116,6 +119,29 @@ func (p *Register) createDirector(proxyDefinition *Definition, balancer Balancer
 			}
 		}
 
+		parameterNames := parameterNameExtractor.Extract(path)
+		for _, parameterName := range parameterNames {
+			parameterValue := router.URLParam(req, parameterName)
+
+			if len(parameterValue) == 0 {
+				log.WithField(
```

Comment:
Here we could:
```go
log.WithFields(log.Fields{
  "paramName": paramName,
  "path": path,
}).Warn("Unable to extract parameter from request")
```

I'm not really sure we should log error and continue with the request, so perhaps warn is more appropriate